Formula/iva.rb

```diff
@@ -60,7 +60,7 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/iva --version 2>&1")
+    assert_match "usage", shell_output("#{bin}/iva 2>&1")
```

Comment:
Looks like the exit status of `iva` with no parameter is 2:
```
root@bd0852548f73:/home/linuxbrew# iva
usage: iva [options] {-f reads_fwd -r reads_rev | --fr reads} <output directory>
iva: error: the following arguments are required: Output directory
root@bd0852548f73:/home/linuxbrew# echo $?
2
```
Try changing to:
```
assert_match "usage", shell_output("#{bin}/iva 2>&1", 2)
```
or:
```
assert_match "usage", shell_output("#{bin}/iva -h 2>&1")
```